heron/tmaster/src/cpp/manager/tmaster.cpp

```diff
@@ -113,7 +108,7 @@ TMaster::TMaster(const std::string& _zk_hostport, const std::string& _topology_n
   tmaster_location_->set_master_port(master_port_);
   tmaster_location_->set_stats_port(stats_port_);
   DCHECK(tmaster_location_->IsInitialized());
-  EstablishTMaster(EventLoop::TIMEOUT_EVENT);
+  EstablishPackingPlan(EventLoop::TIMEOUT_EVENT);
```

Comment:
Fetching the packing plan is required for initialization so we can determine the stream manager id set (`absent_stmgrs_`). We need to be prepared with this info before TMaster registers it's location and starts receiving registration requests from stream managers.